src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,131 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * An object that handles reading and writing of data to save file.
+ *
+ */
+public class Storage {
+
+    private static File savedFile = null;
+
+    /** Constructor for a Storage object */
+    protected Storage() {
+
+    }
+
+    /**
+     * Is called upon starting Duke. Attempts to find a save file.
```

Comment:
Should start with a verb
is called upon starting duke can be left at description
```suggestion
     * Attempts to find a save file.
```